Makefile

```diff
@@ -75,6 +75,19 @@ UFTRACE_LDFLAGS = $(COMMON_LDFLAGS) $(LDFLAGS_$@) $(LDFLAGS_uftrace)
 DEMANGLER_LDFLAGS = $(COMMON_LDFLAGS) $(LDFLAGS_$@) $(LDFLAGS_demangler)
 LIB_LDFLAGS = $(COMMON_LDFLAGS) $(LDFLAGS_$@) $(LDFLAGS_lib) -Wl,--no-undefined
 
+ifeq ($(DEBUG), 1)
+  COMMON_CFLAGS += -O0 -g
+else
+  COMMON_CFLAGS += -O2
```

Comment:
I think we need to keep `-g` for the non-DEBUG case too.

Reply:
Thanks!